### PR TITLE
Enrich app metadata in Chart.yaml

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/19663

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/kyverno/Chart.yaml [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+  language. Kyverno is designed to work nicely with tools you already use like
+  kubectl, kustomize, and Git.
+engine: gotpl
+home: https://github.com/giantswarm/kyverno-app
+icon: https://s.giantswarm.io/app-icons/kyverno/1/light.svg
+keywords:
+  - kubernetes
+  - policy agent
+  - validating webhook
+  - admissions controller
```
Comment (rossf7) [inline helm/kyverno/Chart.yaml:22]: Keywords are from https://github.com/kyverno/kyverno/blob/e5e849acfebd0461ae147d066821fdf86c620fc7/charts/kyverno/Chart.yaml

File: helm/kyverno/Chart.yaml [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
+name: kyverno
+restrictions:
+  clusterSingleton: true
```
Comment (rossf7) [inline helm/kyverno/Chart.yaml:25]: Since we're installed CRDs I think we should only allow installing once.

File: helm/kyverno/Chart.yaml [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
   config.giantswarm.io/version: 1.x.x
 dependencies:
   - name: policy-reporter
     version: 1.9.3
+description: |
+  Kyverno is a policy engine designed for Kubernetes. It can validate, mutate,
+  and generate configurations using admission controls and background scans.
+  Kyverno policies are Kubernetes resources and do not require learning a new
+  language. Kyverno is designed to work nicely with tools you already use like
+  kubectl, kustomize, and Git.
```
Comment (marians) [inline helm/kyverno/Chart.yaml:14]: Appears a bit long to me, but no blocker. Let's see how our UIs deal with a text that long.

**Review by marians — APPROVED** [review]

**Review by MarcelMue — APPROVED** [review]

File: helm/kyverno/Chart.yaml (hunk 3 above)
Comment (rossf7) [reply helm/kyverno/Chart.yaml]: Display was fine but it looked a bit long. I removed the last sentence.

Screen Shot 2021-11-04 at 4 17 59 PM

**Pull request merged**